features/pipeline_parallel/resnet.py

```diff
@@ -50,7 +50,8 @@ def train():
     optimizer = torch.optim.Adam(model.parameters(), lr=1e-3)
 
     # build dataloader
-    train_dataloader, test_dataloader = build_cifar(BATCH_SIZE, padding=4)
+    root = os.environ['DATA']
```

Comment:
I used `root = os.environ.get('DATA', './data')` in other features so that even if `DATA` is not specified, the dataset will be downloaded to the current directory by default.